lib/urbanopt/scenario/default_reports/scenario_report.rb

```diff
@@ -89,12 +89,15 @@ def initialize(hash = {})
             @reporting_periods << ReportingPeriod.new(rp)
           end
 
-          # intialized here to be used in the add_feature_report method
+          # feature_report is intialized here to be used in the add_feature_report method
           @feature_reports = []
           hash[:feature_reports].each do |fr|
             @feature_reports << FeatureReport.new(fr)
           end
 
+          # initialize file_name 
+          @file_name = 'default_scenario_report'
```

Comment:
document that this is the filename without any extension (since you assign the .json and .csv extensions later)